sorc/chgres_cube.fd/static_data.F90

```diff
@@ -1,39 +1,40 @@
- module static_data
+!> @file
+!! @brief Process static surface data
+!!
+!! @author gayno NCEP/EMC
+!!
+!! Abstract: Read pre-computed static/climatological data on the fv3 
+!!    target grid.  Time interpolate if necessary (for example a
+!!    monthly climo field).
+!!
+!! Public Subroutines:
+!! -------------------
+!! - get_static_fields -       Driver routine to read/time interpolate
+!!                             static/climo fields on the fv3 target
+!!                             grid.
+!! - cleanup_static_fields -   Free up memory for fields in this module.
+!!
+!! Public variables:
+!! -----------------
+!! - alnsf_target_grid -                near ir black sky albedo
```

Comment:
This is fine for a first pass, but in a future pass we should move the documentation of the variables to the variables themselves. Then we can remove this list of public variables - doxygen constructs its own list from the code anyway.